### ISSUE TITLE
Artifacts of the output

### ISSUE THREAD

**Issue by jinyixin621** [huangzehao/torch-srgan#11]
Hello,

As I noticed your outputs have some red and green dots pattern which looks like artifacts generated by the network. But from the output of the original paper, there is no such effect. Do you have any idea about this?

Thanks

**Comment by huangzehao** [comment]
Have you update your codes?
https://github.com/huangzehao/torch-srgan/blob/master/test.lua#L34
It is caused by batchnorm, so I set model:evaluate() to model:training().

**Comment by jinyixin621** [comment]
But it says in the paper that "During test time we turn batch-normalization update off to obtain an output that deterministically depends only on the input [32]."
